Add Mountain America Credit Union

parkerbxyz opened the pull request.

parkerbxyz committed:
Add Mountain America Credit Union

jtagcat commented on _data/banking.yml:
```diff
@@ -624,6 +624,14 @@ websites:
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      doc: https://www.macu.com/home/services/online-banking/new-online-branch
```
```suggestion

```

jtagcat reviewed (commented):
The doc doesn't actually give any information other than proof, that it exists.

jtagcat committed:
Remove doc from Mountain America CU

This documentation merely states that MFA is now available, it does not give any additional information.

Co-Authored-By: parkerbxyz <17183625+parkerbxyz@users.noreply.github.com>

kmpoppe commented on _data/banking.yml:
```diff
@@ -630,7 +630,6 @@ websites:
       tfa: Yes
       sms: Yes
       email: Yes
-      doc: https://www.macu.com/home/services/online-banking/new-online-branch
```
The documentation you provided says "Receive MFA PINs through text, email or automated phone call.", so could you please add this option?

kmpoppe replied:
```suggestion
      phone: yes
```

parkerbxyz committed:
Add phone to Mountain America CU
